### DISCUSSION TITLE
请问用armbian-ddbr创建的原版系统备份包在哪里？

### DISCUSSION THREAD

**Discussion by lordshenone** [ophub/amlogic-s9xxx-armbian#2494] [Amlogic]
我是中兴860AV3.2-M的盒子，买来时原来的系统就有root权限。在装armbian的时候用U盘启动后用armbian-ddbr命令创建了原系统的备份到U盘里，备份是成功创建的。装好系统后拔掉U盘装到其他linux机器上想把备份文件拷出来，结果原来备份的/ddbr这个目录里找不到备份的文件。有没有大神知道这个备份文件应该怎么找到？谢谢。

**Comment by ophub** [comment]
U盘中的/ddbr

**Reply by lordshenone** [reply]
可是U盘中的/ddbr这个目录里只有一个install文件夹，下面是空的。难道要用这个U盘启动盒子才能看到，插在别的机器上不行？

**Reply by lordshenone** [reply]
用制作的U盘启动了机器，又执行了一次备份操作，这次备份文件没有消失。但第一次装机备份原来的andriod系统的文件就消失了，并且/ddbr目录下面还有个空的install目录。感觉是不是安装至emmc的脚本文件里有啥bug，为了系统干净的安装把原本不是镜像里的文件都删除了。只是猜测而已，并没有去验证。

不管如何还是感谢大佬做的这个系统，让我们手里落灰的电视盒子发挥余热。

**Comment by ophub** [comment]
你第每次使用armbian-ddbr生成的备份文件都是在/ddbr下。备份好的文件要及时下载转移。

第一次从armbian备份安卓系统生成的备份先下载了再安装。因为安装的时候会清空/ddbr变挂载armbian处理安装的工作。

**Reply by lordshenone** [reply]
好的，明白了，这个注意事项也请添加到项目的说明文件里吧，有可能很多人并不知道。谢谢。

**Reply by lordshenone** [reply]
另外再提一个小小的要求，备份文件能不能自定义文件名，或者加上日期和时间作为文件名。这样可以有多个版本的备份。

**Comment by ophub** [comment]
我今天修改了armbian-install文件，不再删除U盘中的备份文件了。

现在的armbian-ddbr是读写固定路径固定文件的，这样可以减少交互对话让新人更轻松使用。你如果要在一个U盘中备份多个，可以把已有的文件重命名。

**Reply by lordshenone** [reply]
感谢大佬改进。
